flink-libraries/flink-gelly/src/main/java/org/apache/flink/graph/library/HITSAlgorithm.java

```diff
@@ -273,4 +289,23 @@ public void sendMessages(Vertex<K, Tuple2<DoubleValue, DoubleValue>> vertex) {
 			return initVertexValue;
 		}
 	}
+
+	public static class AuthorityEdgeMapper<K, EV> implements MapFunction<Edge<K, EV>, String> {
```

Comment:
A boolean is stored as a byte as would a bitmask which allows the edge set to be compressed on bidirectional edges. Not sure how common bidirectional edges are in real-world datasets.